test/dom5_test.js

```diff
@@ -122,6 +127,24 @@ suite('dom5', function() {
       });
     });
 
+    suite('hasSpaceSeparatedAttributeValue', function() {
+
+      test('returns false for a non-set attribute', function() {
+        var divA = doc.childNodes[1].childNodes[1].childNodes[0];
+        assert.equal(dom5.hasSpaceSeparatedAttributeValue(divA, 'bar', 'b1'), false);
+      });
+
+      test('returns true for a matching attribute value', function() {
+        var divB = doc.childNodes[1].childNodes[1].childNodes[0].childNodes[1];
+        assert.equal(dom5.hasSpaceSeparatedAttributeValue(divB, 'bar', 'b1'), true);
+      });
+
+      test('returns true for a matching space separated value', function() {
```

Comment:
Do you mean that even though inside different suites, test names should be totes unique?  I hadn't thought about that, since I often use suite names to act like namespaces, qualifying test name.  This changes things.  FOREVER.